.eslintrc

```diff
@@ -72,7 +72,7 @@
     "babel/generator-star-spacing": ["error", { before: false, after: true }],
 
     "react/display-name": ["error", { ignoreTranspilerName: true }],
-
+    "react/prefer-stateless-function": ["off"],
```

Comment:
Stateless function components don't have `shouldComponentUpdate` methods by definition. We shouldn't use those unless they don't receive any props or do receive props that *aways* change.